eng/common/scripts/job-matrix/Create-PrJobMatrix.ps1

```diff
@@ -100,12 +100,14 @@ function GeneratePRMatrixForBatch {
     $matrixResults = @()
     foreach ($matrixConfig in $matrixConfigs) {
       Write-Host "Generating config for $($matrixConfig.Path)"
+      $nonSparse = $matrixConfig.PSObject.Properties['NonSparseParameters'] ? $matrixConfig.NonSparseParameters : @()
 
       $matrixResults = @()
       if ($directBatch) {
         $matrixResults = GenerateMatrixForConfig `
           -ConfigPath $matrixConfig.Path `
           -Selection $matrixConfig.Selection `
+          -NonSparseParameters $nonSparse `
```

Comment:
If we change it here, do you think we should also pass the NonSparseParameters through to the `indirect` matrix generation down below?

Reply:
Yep I just forgot to update that :D